ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
```

Comment:
nit: Shouldn't this just be ViewWidth
The contact width and view width are the same.